SIL.Media.Tests/AudioRecorderTests.cs

```diff
@@ -430,6 +431,22 @@ public void Play_DoesPlay ()
 			}
 		}
 
+		[Test]
+		public void Play_DoesPlayMp3()
+		{
+			using (var file = TempFile.FromResource(Resources.finished, ".mp3"))
```

Comment:
Since we are mocking everything no temp file is necessary. Just a path ending in .mp3 should work.